pos_empty_home/i18n/fr.po

```diff
@@ -0,0 +1,28 @@
+# Translation of Odoo Server.
+# This file contains the translation of the following modules:
+# 	* pos_empty_home
+#
+msgid ""
+msgstr ""
+"Project-Id-Version: Odoo Server 9.0c\n"
```

Comment:
Could you update this fr po file for 12.0 serie ?

Reply:
Isn't this generated by travis? :thinking: